pkg/os/util_test.go

```diff
@@ -81,6 +75,10 @@ func TestFileExists(t *testing.T) {
 
 	err = os.Chmod(dirname, 0000)
 	assert.NoError(t, err)
+	defer func() {
+		assert.NoError(t, os.Chmod(dirname, 0777))
+	}()
+
```

Comment:
Not clear if this change is related to this commit?

Reply:
This change is required because we have changed the permission of `dirname` to `000`. We need to revert it back for `t.TempDir` to perform the cleanup successfully.

https://github.com/crc-org/crc/blob/1425c16d6ca9ae043f4ddcb817f7317443aa70d2/pkg/os/util_test.go#L82-L83